### ISSUE TITLE
Different outlook browsers rendering differently?

### ISSUE THREAD

**Issue by scr1ptk1d** [mjmlio/mjml#2897]
**Describe the bug**
Hello, I have made an mjml signature. I'm dismayed to find it renders differently (and wrongly) in different outlook browsers. I grouped my columns so they would not split, and tried adding a owa="desktop" so it wouldn't try to stack like it's mobile. Mobile loaded properly prior to the grouping, now it is not, sticking with grouping because the windows version was _much_ worse there. I'm also at only 80% column width which I see is a common issue.

**To Reproduce**
1. Render with [https://mjml.io/try-it-live/d3oDF3_SZh](https://mjml.io/try-it-live/d3oDF3_SZh)
3. Send to an outlook address where they use windows app.

**Expected behavior**

![image](https://github.com/user-attachments/assets/d92760d9-c4f4-4ab5-b92c-0e1efe939aaa)

**MJML environment (please complete the following information):**
 - 6.10.7-arch1-1 (64-bit)
 - MJML Version unsure where to find
 - https://mjml.io/ and VSCODE MJML Editor v1.1.1

**Email sending environment(for rendering issues)**:
 - Proton in Firefox.

**Affected email clients (for rendering issues):**
 - Outlook
 - Windows

**Screenshots**
Windows:
![image](https://github.com/user-attachments/assets/65f18028-9a07-4886-9b0c-123aa5d792a1)
Mobile:
![Screenshot_2024-09-05-15-02-13-80_45e686c594768066ad9911d54d96f72b](https://github.com/user-attachments/assets/4c84b698-6dc2-4f89-8abe-e180c4c5d7a5)

**Comment by iRyusa** [comment]
Try send your email with https://putsmail.com/ and not proton as it will alter the HTML. Keep in mind that MJML isn't really meant to be used for email signature as you can't push any head style from signature. It's best to stick with table &  inline style for signature.

**Comment by scr1ptk1d** [comment]
> Try send your email with https://putsmail.com/ and not proton as it will alter the HTML. Keep in mind that MJML isn't really meant to be used for email signature as you can't push any head style from signature. It's best to stick with table & inline style for signature.

Hello iRyusa, I tried dropping my MJML into putsmail, and it just printed text like "Thanks, First Last Job Title Location Ph:" etc. Do I have to use straight html with putsmail instead of MJML? I'd had a similar issue with regular HTML and someone on reddit here suggested I use MJML instead: [https://www.reddit.com/r/HTML/comments/1es9fwu/html_noob_trying_to_make_an_email_signature_lines/](https://www.reddit.com/r/HTML/comments/1es9fwu/html_noob_trying_to_make_an_email_signature_lines/)
Could you please share an example of table & inline? This is just a hobby so I'm not exactly sure what I'm doing.

**Comment by iRyusa** [comment]
No client would "read" MJML, you need to send the HTML

**Comment by scr1ptk1d** [comment]
> No client would "read" MJML, you need to send the HTML

Okay, I was a tad frustrated you were saying to use HTML because I'd already tried that, until I realized you were saying to _convert_ it to HTML. So I ran yay -S nodejs-mjml and mjml example.mjml -o example.html and pasted in the rendering into proton and it worked! Mobile still stacked, but I got input from peers that it looks better that way anyway so I'm rolling with it. Thanks!
